include/deal.II/distributed/tria_base.h

```diff
@@ -98,6 +98,11 @@ namespace parallel
 
     /**
      * Implementation of the same function as in the base class.
+     *
+     * @note this functions copies the cells, but not the communicator, 
```

Comment:
```suggestion
     * @note This functions copies the cells, but not the communicator, 
```

Reply:
There is also an extra `s` add the end of `function`